DigitalLearningSolutions.Web.Tests/Services/UserServiceTests.cs

```diff
@@ -388,7 +388,7 @@ public void SetCentreEmails_calls_UserDataService_SetCentreEmail_for_each_item_i
         }
 
         [Test]
-        public void SetCentreEmails_calls_UserDataService_does_nothing_if_given_an_empty_dictionary()
+        public void SetCentreEmails_calls_UserDataService_does_not_call_data_service_if_given_an_empty_dictionary()

```

Comment:
I think you still need to remove the "calls_UserDataService" part

Reply:
Oh, duh!